backend/src/paymaster/index.ts

```
@@ -237,6 +243,38 @@ export class Paymaster {
+      const paymasterData = hexConcat([
+        '0x00',
+        defaultAbiCoder.encode(
+          ['uint48', 'uint48', 'address', 'address', 'uint256', 'uint32'],
+          [validUntil, validAfter, feeToken, ethers.constants.AddressZero, ethPrice, priceMarkup]
+        ),
+        sig,
+      ]);
+
+      return paymasterData;
+    } catch (err: any) {
+      throw new Error('Failed ' + err.message)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)